shuttle.yaml

```diff
@@ -9,6 +9,7 @@ scripts:
     description: build code
     actions:
       - shell: go build
+      - shell: sleep 2
```

Comment:
remove?

Reply:
Some debug stuff I did, for testing longer durations in humio

Will remove xD